client/common/sass/components/_filters.sass

```diff
@@ -127,3 +127,31 @@
 			.filters__suggestion-tooltip
 				display: inline
 				margin-left: 0.5rem
+
+.filters-summary
+	margin: 0
+	padding: 0
+
+	+desktop-up
+		&:not(:empty)
+			margin-bottom: 32px
+
+	+tablet-down
+		margin-bottom: 8px
+
+	li
+		display: inline-block
+		margin: 2px 2px 2px 0
```

Comment:
Small knitpick but according to the figma, the margin between each element is 8px